deploy/sre-prometheus/100-dedicated-admin-operator.PrometheusRule.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-dedicated-admin-operator-alerts
+    role: alert-rules
+  name: sre-dedicated-admin-operator-alerts
```

Comment:
The prometheusRule deployed with d-a-o to monitor blacklisted namespaces is called `sre-dedicated-admin-alerts`
(https://github.com/openshift/dedicated-admin-operator/blob/master/build/templates/olm-artifacts-template.yaml.tmpl)

Wont this `sre-dedicated-admin-operator-alerts` create some confusion?

Reply:
What about renaming this one to `sre-dedicated-admin-operator-offline-alerts` ?

Reply:
You updated the Group, shouldn't we do the same with `metadata.Name` ?